src/account/Account.ts

```diff
@@ -98,4 +100,77 @@ export class Account implements IAccount {
       );
     });
   }
+
+  /**
+   * This method aims to buy rolls on behalf of the sender.
+   *
+   * @param amount - The amount of rolls to be purchased
+   * @param fee - The fee to be paid for the transaction.
+   * @returns An ITransactionDetails object. It contains the operationId on the network.
+   */
+  public async buyRolls(
+    amount: string,
```

Comment:
Should be an unsigned integer if that exists

Reply:
we dont have that in typescript, hence a bigint would suffice here. I could also add a check > 0 if you wish ?